riscv/src/lib.rs

```diff
@@ -29,21 +29,52 @@ pub mod runtime_32;
 static TARGET_STD: &str = "riscv32im-risc0-zkvm-elf";
 static TARGET_NO_STD: &str = "riscv32imac-unknown-none-elf";
 
+#[derive(Clone)]
+pub struct CompilerOptions {
+    pub field: KnownField,
+    pub runtime: RuntimeEnum,
```

Comment:
My idea for the Options here was to avoid having this `RuntimeEnum`, or even knowing there is a "runtime" at the time you use it.
Only pass the minimum amount of information here needed to instantiate the correct runtime at the latest time possible (`translate|compile`), to avoid needing a runtime enum.
Doesn't this work?

Reply:
e.g., from the `KnownField` you know whether to instantiate the 16 or 32 bit runtime

Reply:
Yes I'll do it that way, but it was easier to keep `RuntimeEnum` for now to make the tests work, then I'll iteratively remove the RuntimeEnum from this struct too.

Reply:
@pacheco please check now, I think it looks nice. RuntimeEnum is completely gone.